python/private/toolchains_repo.bzl

```diff
@@ -78,6 +81,7 @@ py_toolchain_suite(
             python_version = python_version,
         )
         for platform, meta in PLATFORMS.items()
+        if platform in loaded_platforms
```

Comment:
TODO: maybe `platforms` should only contain the keys that are needed here. We should accept a parameter instead of using the global variable `PLATFORMS`.